app/views/checkouts/index.html.erb

```diff
@@ -3,3 +3,17 @@
 <h1>Checkouts</h1>
 
 <%= render 'shared/navigation' %>
+
+<% @checkouts.each do |checkout| %>
+  <div class="row">
+    <%= checkout['fields']['status']%>
```

Comment:
It looks like this isn't accessible from the `Checkout` class.  It seems like there are accessor in that class for each of these pieces of data.  Perhaps you for forgot to push up a commit where you changes this view from accessing the json directly to model accessors?

Reply:
Thanks, it looks like I dropped a commit.